build.sbt

```diff
@@ -27,6 +27,11 @@ val `scala2.12` = Seq(
   crossScalaVersions := Seq("2.11.8", "2.12.1")
 )
 
+val `scala2.12_full` = Seq(
```

Comment:
We should probably adopt a different naming convention. What about `scala 2.11`, `scala 2.11 to 2.12` and `scala 2.10 to 2.12`?